### DISCUSSION TITLE
Create New Codespace: Error retrieving available machine types.

### DISCUSSION THREAD

**Discussion by jozkee** [community/community#11145] [Codespaces]
I've been having issues with trying to load a codespace from another user PR.
Steps: 
1. open the PR.
2. Hit "shift + ." to open github.dev in a new tab.
3. Open terminal and click Continue working on...
4. Select: Create New codespace.

I get this error:
```
Create New Codespace: Error retrieving available machine types. 
Status code: 400. Correlation ID: F206:4712:317998:33C264:62019272
```

Now more recently, I've been having this other error by just hitting "." (if I should start 1 discussion per error let me know):
```
Sign in failed: Error: API rate limit exceeded for user ID 16040868.

Source: GitHub Authentication (Extension)
```
I already tried disbaling/enabling the extension and logging out and logging back in.

cc @eerhardt

**Comment by canokcelik** [comment]
Codespace is a paid service for now. Have you been involved a team or organisation?

You may consider to check out following link: https://github.com/features/codespaces

**Comment by D1no** [comment]
<sub>Fund this via google by searching `"Sign in failed: Error: API rate limit exceeded for user"`</sub>

Same here, couldn't use copilot because GitHub Authentication doesn't work. Nothing to do with codespaces @canokcelik 

![image](https://user-images.githubusercontent.com/2397125/223324913-c37e38c5-3846-4a24-860e-e39ea6f5511d.png)


https://user-images.githubusercontent.com/2397125/223324934-8352cdcd-6cfc-4c3d-b632-b00d571422df.mp4

**Comment by CrazyCat7100** [comment]
Create New Codespace: List devcontainer definitions request failed with status 400. Correlation ID: D15E:3C35:ACAF92:1660C3C:64D05254. My error. can somebody help?
